### PR TITLE
Vilukissa

### PR DESCRIPTION
**Pull request by vilukissa68**
Sorry I lied last code was bad :(

But now it's good I promise

### CODE REVIEW

File: rpi/poller.py [hunk 1]
```diff
@@ -51,4 +62,6 @@ def beep():
         GPIO.output(buzzer, GPIO.LOW)
 
 
-setInterval(beep, 10)
+# Prevent auto run on import
+if __name__ == "__main__":
```
Comment (sjaks) [inline rpi/poller.py:66]: What is this???

File: rpi/poller.py [hunk 2]
```diff
@@ -22,6 +22,17 @@
 ENTRY = "https://jaks.fi/wakey/get"
 
 
+# Test your local speaker system
+# If you hear a sound everything works
+def speakerTest():
```
Comment (sjaks) [inline rpi/poller.py]: Vey cool

**Review by sjaks — CHANGES_REQUESTED** [review]
Also there's a conflict, plz solve :D

File: rpi/poller.py (hunk 1 above)
Comment (vilukissa68) [reply rpi/poller.py:66]: import poller
doesn't work as expected this fixes it

File: rpi/poller.py (hunk 2 above)
Comment (vilukissa68) [reply rpi/poller.py]: ty

**Comment by vilukissa68** [comment]
Is fix now. Plz accept so I can pay my bills :money_mouth_face:

**Pull request merged**